sap/5.9/modules/ROOT/pages/sap-connector-reference.adoc

```diff
@@ -312,6 +314,49 @@ tRFC can be serialized using inbound and outbound queues, hence the name queued
 * SAP:SYSTEM_BUSY
 * SAP:TIMEOUT
 
+[[asynchronousRemoteFunctionCall]]
+=== Asynchronous RFC
```

Comment:
I'm concerned about the naming. Why `async-rfc` translates to `Asynchronous Remote Function Call` and `asynchronousRemoteFunctionCall` into `Asynchronous RFC`? Shouldn't it be the other way around?

Reply:
This is accurate as verified on the Slack channel, not sure why though